salt/elastic_stack/elasticsearch/apps/cronjobs.sls

```diff
@@ -0,0 +1,28 @@
+
+ensure_installation_of_jq:
+  pkg.installed:
+    - name: jq
+
+manage_search_index_pruning_job:
+  cron.present:
+    - identifier: PRUNE_SEARCH_INDICES
+    - user: root
+    - hour: random
+    - minute: random
+    - name: >-
+       for index in `curl -s 'localhost:9200/_aliases' | jq 'to_entries | \
```

Comment:
The `\` shouldn't be necessary as the `>-` operator will reconstitute the string to a single line.

Reply:
Oh, right. Have just fixed that.